### PR TITLE
Add comment in README to point out static classes

### PR DESCRIPTION
**Pull request by RobGibbens**
Mapping classes are provided. This PR points out the option to use static typing instead of XAML strings, as an alternative.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -67,6 +67,21 @@ Using it in your XAML couldn't be easier:
...
+```xml
+<ContentPage xmlns="http://xamarin.com/schemas/2014/forms"
+             xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
+             xmlns:ap="http://avantipoint.com/mobiletoolkit"
+             x:Class="SampleFonts.MainPage">
+  <StackLayout>
+    <Label Text="{ap:Icon IconName={x:Static ap:FontAwesomeRegular.User}}" />
+    <Button Text="{ap:Icon IconName={x:Static ap:FontAwesomeRegular.User}}" />
+    <Image Source="{FontImage Glyph={x:Static ap:FontAwesomeRegular.CheckCircle}, Color=Blue, Size=60}" />
+  </StackLayout>
```
Comment (dansiegel) [inline README.md:81]: The mapping classes aren't really meant to be used quite like this as they just point to the actual unicode character so you would need to set the Font Family as well...

**Comment by dansiegel** [comment]
This is actually in the sample though for some reason it's missing the reference to the FontFamily property off of the Mapping Class. Good catch... appreciate the PR @RobGibbens 

https://github.com/AvantiPoint/AP.MobileToolkit.Fonts/blob/a6ddcbc8ab3ed3153f992d4602f42f22bc5613a6/sample/SampleFonts/SampleFonts/MainPage.xaml#L78-L79

**Comment by dansiegel** [comment]
I'm going to close this PR and create an issue for this. Please correct me if I'm wrong here, but as I thought some more on this, it would seem that your impression was that the mapping classes would map FontAwesomeRegular.User to `far fa-user` and not the unicode character. I think this would actually be a huge enhancement, and I'm not at all against it... It would require building a code generator but that shouldn't be terribly difficult since we're already scanning the css for Font Awesome.

**Comment by RobGibbens** [comment]
Sounds great @dansiegel . Thanks. I just know that I'll never remember all of the shorthand icon names, and I'd use IntelliSense all the time, especially as more fonts get added.  For completeness, the Issue is here : https://github.com/AvantiPoint/AP.MobileToolkit.Fonts/issues/5

**Comment by dansiegel** [comment]
perfect... thanks for confirming that was the issue.

**Pull request closed without merging**